validator/build.gradle

```diff
@@ -14,11 +14,11 @@ repositories {
 }
 
 dependencies {
-    compile group: 'org.topbraid', name:'shacl', version:'1.3.1'
-    compile group: 'org.projectlombok', name:'lombok', version:'1.18.4'
-    compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
+    implementation group: 'org.topbraid', name:'shacl', version:'1.3.1'
+    implementation group: 'org.projectlombok', name:'lombok', version:'1.18.4'
+    implementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
     annotationProcessor 'org.projectlombok:lombok:1.18.12'
 
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
-    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.6.0'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.6.0'
```

Comment:
tomuhle nerozumím, ale jestli je to ok, tak je to ok

Reply:
to je jen nova syntax gradle